### PR TITLE
Portfolio: Details-Pop-Up-Window

### PR DESCRIPTION
**Pull request by maximoortelli**
_Portfolio-details-popup-window is an exercise in which I will add new pop up windows to my cards to my portfolio, both for desktop and mobile versions. The goal here is to master all of the tools and best practices learned about in previous steps to use them in all Microverse projects and most likely in my future job as well._

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+const multiPostSories = new ProjectContructor(
+  'media/Snapshoot Portfolio desk 1.png',
+  'Multi-Post Stories',
+  { name: 'FACEBOOK', position: 'Full Stack Dev', date: '2015' },
+  'Experimental content creation feature that allows users to <br> add to an existing story over the course of a day without spamming their friends.',
+  ['HTML', 'Ruby on rails', 'CSS', 'Javascript'],
+  'multi-post-stories',
+);
+
```
Comment (rotimiazeez) [inline js/app.js:32]: Please note that your desktop popup is a bit off from the original design. Kindly adjust the design a bit for a better UI 👍🏽
Figma Design | Your Design
Screenshot from 2023-07-26 18-35-05 | Screenshot from 2023-07-26 18-39-47

Comment (rotimiazeez) [reply js/app.js:32]: Please note that the icon on the `see source ` button is currently broken, kindly fix it. 😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- The projects section is created dynamically with the information from the projects array 👏🏽
- Good teamwork 👌🏽
- Good App UI 💪🏽 
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: js/app.js (hunk 1 above)
Comment (maximoortelli) [reply js/app.js:32]: Done👍

File: pop.css [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(10px);
+}
```
Comment (Strangeal) [inline pop.css:14]: - [x] Great job in implementing the changes suggested by the previous reviewer. However, I noticed that after fixing the issues you had with `image` pointed out by the previous reviewer it introduced a few issues which I will highlight in the below comment:
   - Your `see live` and `see source` buttons are not visible on large screen sizes
   -  Your project description `font-size` becomes very small on a large screen size and does match that of the `Figma` design.   
   
- Kindly consider fixing the issues mentioned above to help make your design look responsive, match that of the `Figma` design, and create a good user experience.  Below is a short screen recording of what the issue looks like:



https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/107192076/8c868266-a522-40f4-8f92-61e897236d94

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @maximoortelli, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: pop.css (hunk 2 above)
Comment (maximoortelli) [reply pop.css:14]: Thank you very much for helping me, for taking the time and providing clear examples of how to solve it.
All the changes that you have suggested to me as have been corrected satisfactorily 👍

File: js/app.js [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+              <div class="flex-row button-container">
+                 <div class="grupo-boton">
+                   <a href="https://maximoortelli.github.io/Portfolio-setup-and-mobile--first/" target="_blank">
+                       <button class="flex-row">See live <img class="button-icon" src="media/See live icon.png"></button>
+                   </a>
+                 </div>
+                 <div class="grupo-boton">
+                   <a href="https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/tree/Portfolio-Pop-Up-Window" target="_blank">
+                     <button class="flex-row">See source <img class="button-icon" src="media/See source icon.png"></button>
+                   </a>
```
Comment (shahadat3669) [inline js/app.js:216]: - [x] 👏🌟 Great job on making progress with updating the code based on the previous reviewer's request! I can now see the buttons on the desktop version, which is fantastic! However, there are still some adjustments that need to be made to meet the design requirements. The current button styles don't match the design, so I suggest paying close attention to the design and updating the code accordingly.

image

By fine-tuning these button styles and adhering closely to the design, your project will look even more polished and professional! 🎉💪 Keep up the good work, and happy coding! 💻🚀

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi Everyone, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ No linter Errors
✅ Descriptive PR Title
✅ Descriptive PR Description



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table  width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/app.js (hunk 3 above)
Comment (maximoortelli) [reply js/app.js:216]: Thank you very much for advising me, I hope that now with all these changes you can approve me. I have completed all revisions satisfactorily 👍

**Review by ademibowale — APPROVED** [review]
Hi @roshanasingh4 , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- The popup follows the design for both desktop and mobile versions ✔️ 
- When the user clicks (or taps) the close (X) button, the popup disappears.✔️
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**